pkg/api/types.go

```diff
@@ -1225,6 +1289,8 @@ type PodSpec struct {
 	// If specified, these secrets will be passed to individual puller implementations for them to use.  For example,
 	// in the case of docker, only DockerConfig type secrets are honored.
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	// Multiple tolerations with the same key are allowed.
+	Tolerations []Toleration `json:"tolerations,omitempty"`
```

Comment:
Oh, sorry I wasn't clear about this -- for first version, taints should be alpha annotation on Node, and tolerations should be alpha annotation on Pod. We can move to fields of NodeSpec and PodSpec in next version.